### ISSUE TITLE
NZBGet 20

### ISSUE THREAD

**Issue by woiza** [nzbget/nzbget#522]
Not an issue, just out of curiosity. Are you going to release v20.0?
20.0-testing-r2159 works perfectly for me, but there are many convenient docker images relying on your latest release tag.

**Comment by hugbug** [comment]
I have a crash issue to investigate. Don't want to publish v20 as stable knowing it crashes sometimes. At least not until I did my best trying to resolve the issue.

Don't all these docker images use official nzbget installer with built-in update feature? In that case you can update to the latest testing from webui.

**Comment by woiza** [comment]
Yes they do and the update feature works fine.

**Comment by JackDandy** [comment]
Hi @hugbug 

**_If_** by "crash" you are referring to the crash fixed @ https://github.com/nzbget/nzbget/issues/501 then I can confirm that... 
* I suffered the same random crashing
* I don't have any more crashes since using a build with [fix 501](https://github.com/nzbget/nzbget/commit/d93769021a8df008017de8ed1058356574781a79)

Hope that helps.

**Comment by hugbug** [comment]
@JackDandy, thanks, good to know that the fix has helped.

I have a few more reports (from the same user) when using the build with that fix. There must be another issue. It seems to be very rare, so maybe I should publish v20 nonetheless as it seems to be stable enough for most users.

**Comment by JackDandy** [comment]
@hugbug Yup, I reckon the issue has stalled things long enough and a new release that fixes things for most users would be good. Those in the shadows may come forward and report if they feel to.  At the very least you could call it testing-2160 or release candidate 1. Either way, i'd release that crash fix asap for those that are not able to compile themselves - it's kinda too important a fix to not have out there.

**Comment by hugbug** [comment]
I've got another crash dump from the user. After analysing it I have finally got an idea of what was causing the crash - see #529.

V20 stable can be releases after this is fixed (and tested in a testing release).

Although the bug isn't specific to v20 and exists since v19.

**Comment by hugbug** [comment]
Closing this as resolved.
